spec/lib/fake_data/complaint_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe FakeData::Complaint do
+  let(:complaint) { FakeData::Complaint.new }
+
+  describe "#data" do
+    it "returns the data for a single complaint" do
+      data = complaint.data
+
+      expect(data[:id]).to_not be_nil
+      expect(data[:id]).to be_a(String)
+      expect(data[:attributes]).to be_a(Hash)
+      expect(data.dig(:relationships, :grantAward, :meta, :id)).to be_a(String)
+      expect(data[:links]).to be_a(Hash)
+    end
+  end
+
+  describe "datetime" do
+    it "returns a string formatted as ISO-8601 UTC datetime" do
+      expect(complaint.datetime).to be_a(String)
+      expect(Time.iso8601(complaint.datetime)).to be_a(Time)
```

Comment:
[suggestion] `complaint.datetime` is not memoized (and can't be, given how it's used), so lines 20 & 21 are testing different values. I think it would be good to either have a line here that stores the datetime and `expect` against it. Or break these into two differently named tests.